Code-4-Community/frontend-scaffold
# Update antd and styled components

## Why

We arent doing routing, antd styling, or styled components!

## This PR

Adding craco antd, theming, and proper examples of styled components (replacing all instances of .less with equivalent styled component code). Also fixed routing issues where pages would be reloaded or history not pushed when navigating. Also added example navbar.


## Verification Steps

Used npm run start and interacted with the site to ensure css was working as before and routing was not reloading the pages and was saving history to the browser.
